airbyte-integrations/connectors/destination-redshift/src/test-integration/java/io/airbyte/integrations/destination/redshift/typing_deduping/RedshiftStandardInsertsTypingDedupingTest.java

```diff
@@ -16,4 +27,49 @@ protected ObjectNode getBaseConfig() {
     return (ObjectNode) Jsons.deserialize(IOs.readFile(Path.of("secrets/1s1t_config.json")));
   }
 
+  @Test
+  public void testStandardInsertBatchSizeGtThan16Mb() throws Exception {
```

Comment:
should we actually move this into the base TD test class? Seems useful for all destinations to verify handling of nontrivial data size

Reply:
Do we have limitations in staging batch size u recall. put it in std insert flow for SQL stmt limit. probably can pull it up to AbstractRedshiftTD too.

Reply:
:shrug:

we probably target a specific size, but I'm not aware of redshift (or any staging dest) setting a hard limit on file size (and we're probably not reaching that limit anyway...)